invenio_records_resources/services/files/tasks.py

```diff
@@ -22,7 +22,8 @@ def fetch_file(service_id, record_id, file_key):
     file_record = service.read_file_metadata(system_identity, record_id, file_key)
     source_url = file_record.data["uri"]
     # download file
-    with requests.get(source_url, stream=True) as response:
+    # verify=True for self signed certificates by default
```

Comment:
minor: I'm not sure I understand this one. If `verify=True`, that means that certificates (including self-signed) will be attempted to be verified, which might fail. I'm not suggesting to add `verify=False`, but maybe we can add a TODO (or open an issue) to make all these extra parameters configurable via e.g. `RECORDS_RESOURCES_FILES_FETCH_REQUESTS_ARGS = { ... }`.

Reply:
Yes, by default self signed will fail. I can open an issue to make them configurable.